### DISCUSSION TITLE
구글 api 연동 이메일 넘겨드립니다

### DISCUSSION THREAD

**Discussion by thiat1998** [DevAnyKR/StreamDock.Plugin.GoogleAPI#6] [Q&A]
@gmail.com
부탁드립니다!

**Comment by DevAnyKR** [comment]
이메일 주소로 파일을 첨부하였습니다. 확인해보세요~

**Reply by thiat1998** [reply]
감사드립니다~!

**Reply by DevAnyKR** [reply]
테스트 결과를 알려주시면 감사하겠습니다. - 정상 조회 여부

**Reply by thiat1998** [reply]
> 테스트 결과를 알려주시면 감사하겠습니다. - 정상 조회 여부

제가 착각해서 gmail 기능도 사용하려면 요청 드려야하는줄 알고 요청 드렸었네요.
그래도 파일 추가후 애드센스 부분이 이전과 다르게 활성화 된걸로 확인하였습니다.

**Reply by DevAnyKR** [reply]
구글 계정은 한 번만 연결하면 프로젝트에 연결된 모든 API를 사용할 수 있습니다.

지금은 애드센스 기능만 테스트 가능합니다.
캘린더 기능은 작업 중입니다.

**Reply by thiat1998** [reply]
감사드립니다! 검색하다가 개발자님 블로그도 잘보았습니다~! 기대가되는 프로젝트입니다!
